hoomd/md/minimize/fire.py

```diff
@@ -0,0 +1,206 @@
+# coding: utf-8
+
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+# Maintainer: joaander / All Developers are free to add commands for new
+# features
+
+"""Energy minimizer."""
+
+import hoomd
+
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.data import syncedlist
+from hoomd.data.typeconverter import OnlyFrom, OnlyTypes
+from hoomd.logging import log
+from hoomd.md import _md
+from hoomd.md.integrate import _DynamicIntegrator, _preprocess_aniso
+
+
+def _raise_if_nonpositive(value):
```

Comment:
Use one of the existing validators, such as `nonnegative_real` or `positive_real`.